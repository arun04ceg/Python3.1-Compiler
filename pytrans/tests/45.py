x = False
y = False
z = 3
if x:
 print(x)
elif y:
 print(y)
else:
 print(z)

