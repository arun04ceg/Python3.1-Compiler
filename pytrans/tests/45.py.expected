(program
 (define x (void))
 (define y (void))
 (define z (void))
 (set-global! x #f)
 (set-global! y #f)
 (set-global! z 3)
 (cond
  ((get-global x) (let () (py-print (get-global x))))
  ((get-global y) (let () (py-print (get-global y))))
  (else (let () (py-print (get-global z))))))
